workflowhelpers/internal/space.go

```diff
@@ -111,9 +120,16 @@ func (ts *TestSpace) Create() {
 }
 
 func (ts *TestSpace) Destroy() {
-	deleteOrg := internal.Cf(ts.CommandStarter, "delete-org", "-f", ts.organizationName)
-	EventuallyWithOffset(1, deleteOrg, ts.Timeout).Should(Exit(0))
-
+	if ts.isExistingOrganization {
+		setDefaultQuota := internal.Cf(ts.CommandStarter, "set-quota", ts.organizationName, "default")
```

Comment:
I see.  Yeah, it was primarily the assumption of "default" that worries me.  I think we have to solve that somehow before we can accept this, right?